lib/config.js

```diff
@@ -70,12 +68,16 @@ internals.schemas.options = Joi.object({
         }
     }),
     routes: Joi.object({
-        include: Joi.array().items(Joi.alternatives().try(Joi.object().type(RegExp), Joi.string())), // May register no routes...
-        exclude: Joi.array().items(Joi.alternatives().try(Joi.object().type(RegExp), Joi.string()))
-    })
+        include: Joi.array().items(
+            Joi.alternatives().try(Joi.object().type(RegExp), Joi.string())
+        ), // May register no routes...
+        exclude: Joi.array().items(
+            Joi.alternatives().try(Joi.object().type(RegExp), Joi.string())
+        )
+    }),
+    zeroIndex: Joi.boolean().required()
```

Comment:
This should not be required if there is a default value

Reply:
Updated in a recent commit.